test/unit/variable-scope.test.js

```diff
@@ -631,6 +631,194 @@ describe('VariableScope', function () {
                 expect(scope.values.count()).to.equal(0);
             });
         });
+
+        describe('replaceIn', function () {
+            it('should handle all inputs', function () {
+                var scope = new VariableScope({
+                    values: [{
+                        key: 'var1',
+                        value: 'value1'
+                    }]
+                });
+                expect(scope.replaceIn('{{var1}}')).to.equal('value1');
+                expect(scope.replaceIn(null)).to.equal(null);
+                expect(scope.replaceIn(undefined)).to.equal(undefined);
+                expect(scope.replaceIn(true)).to.equal(true);
+                expect(scope.replaceIn({})).to.eql({});
+                expect(scope.replaceIn('`console.error()`')).to.equal('`console.error()`');
```

Comment:
Also, add Array and Function type

Reply:
They are in separate tests. Do you want them in this test itself?

Reply:
Yeah, not required.